benchmarks/src/main/scala/akka.kafka.benchmarks/ReactiveKafkaProducerBenchmarks.scala

```diff
@@ -19,23 +19,35 @@ import scala.language.postfixOps
 
 object ReactiveKafkaProducerBenchmarks extends LazyLogging {
   val streamingTimeout = 30 minutes
+  val logStep = 100000
+
   type Fixture = ReactiveKafkaConsumerTestFixture[CommittableMessage[Array[Byte], String]]
 
   /**
    * Iterates over N lazily-generated elements and passes them through a Kafka flow.
    */
   def plainFlow(fixture: ReactiveKafkaProducerTestFixture[Int], meter: Meter)(implicit mat: Materializer): Unit = {
     logger.debug("Creating and starting a stream")
+    var lastPartStart = System.nanoTime()
     val future = Source(Stream.from(0, 1))
       .take(fixture.msgCount.toLong)
       .map(number => Message(new ProducerRecord[Array[Byte], String](fixture.topic, number.toString), number))
       .via(fixture.flow)
       .map {
-        msg => meter.mark(); msg
+        msg =>
+          meter.mark()
+          if (msg.offset % logStep == 0) {
+            val lastPartEnd = System.nanoTime()
+            val took = (lastPartEnd - lastPartStart).nanos
+            logger.info(s"Sent ${msg.offset}, took ${took.toMillis} ms to send last $logStep")
+            lastPartStart = lastPartEnd
```

Comment:
I probably shouldn't be closing over an externally declared variable here, right?